recipe/meta.yaml

```diff
@@ -44,7 +44,8 @@ requirements:
     - six >=1.7.3
     - {{ pin_compatible('numpy') }}
     - scipy >=0.17
-    - matplotlib >=2.0.0
+    - matplotlib-base >=3.0.2  # [py3k]
+    - matplotlib      >=2.0.0  # [py2k]
```

Comment:
Should we relax this to just require 2 for both?

Reply:
yes! Man that PR came out of nowhere and from the previous discussions it seemed matplotlib 2.x wasn't going to be split.